many-block/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "many-block"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+chrono = "0.4.19"
```

Please update to `>=0.4.20`.

`0.4.19` is impacted by https://rustsec.org/advisories/RUSTSEC-2020-0159

```suggestion
chrono = "0.4.22"
```